modules/govuk/manifests/node/s_asset_base.pp

```diff
@@ -193,15 +193,10 @@
         mode    => '0755',
       }
 
-      # FIXME: remove after this has been deployed
-      file { '/etc/cron.daily/push_attachments_to_s3':
-        ensure => absent,
-      }
-
       cron { 'push_attachments_to_s3':
         command => '/usr/bin/setlock -n /var/run/virus_scan/push-attachments.lock /usr/local/bin/push_attachments_to_s3.sh /mnt/uploads',
         user    => 'assets',
-        hour    => 6,
+        hour    => 21,
```

Comment:
There is a full sync from the asset master that happens in the early hours of the morning and I didn't want it to clash with that. The less other maintenance tasks it clashes with the better I reckon. I don't think it'll be very unlikely to ever ring us for oncall, at least, I'd be very upset if it did :)